### PR TITLE
feat: generate SDKs for Looker 24.4

### PR DESCRIPTION
**Pull request by jeremytchang**
Release-As: 24.4.0

### CODE REVIEW

File: csharp/sdk/4.0/models.cs [hunk 1]
```diff
@@ -2608,24 +2608,9 @@ public enum InvestigativeContentType
...
-  public bool has_totals { get; set; }
-  /// <summary>If the query results hit the maximum column limit and additional columns were truncated (read-only)</summary>
-  public string columns_truncated { get; set; } = "";
-  /// <summary>Filter expression applied to the query results (read-only)</summary>
-  public string filter_expression { get; set; } = "";
-  /// <summary>Filters applied to the query results (read-only)</summary>
-  public StringDictionary<string> filters { get; set; } = null;
-  /// <summary>Raw sql query. Null if user does not have permission to view sql (read-only)</summary>
-  public string sql { get; set; } = "";
+  public JsonBiMetadata metadata { get; set; } = null;
```
Comment (jkaster) [inline csharp/sdk/4.0/models.cs:2611]: I guess we'll find out if customers complain about this breaking change.

File: csharp/sdk/4.0/models.cs [hunk 2]
```diff
@@ -2608,24 +2608,9 @@ public enum InvestigativeContentType
...
   /// <summary>Json query results (read-only)</summary>
-  public string[] data { get; set; } = null;
+  public string[] rows { get; set; } = null;
```
Comment (jkaster) [inline csharp/sdk/4.0/models.cs:2613]: and another breaking change.

**Review by drstrangelooker — APPROVED** [review]
Looks LGTM To Me

**Review by jkaster — COMMENTED** [review]
Just a comment for now to see if we're going to accept this breaking change for API 4.0

File: csharp/sdk/4.0/models.cs (hunk 1 above)
Comment (jeremytchang) [reply csharp/sdk/4.0/models.cs:2611]: Yeah i noticed this. We need to discuss this internally later on how preventing or mitigating breaking API changes.

**Comment by jeremytchang** [comment]
We've determined the previously noted breaking changes are not breaking.

**Pull request merged**